src/balancer/upstream_peer.rs

```diff
@@ -161,6 +181,11 @@ mod tests {
             "test_agent".to_string(),
             Some("test_name".to_string()),
             None,
+            None,
+            None,
+            None,
+            None,
+            None,
```

Comment:
I don't like this a lot (that column of None). Maybe we should create StatusUpdate manually so at least we see what values are assigned to what parameters? That `StatusUpdate::new` looks like can be a source of bugs (it will be a column of booleans and None)